### ISSUE TITLE
Error on Collection creation

### ISSUE THREAD

**Issue by iamvaliyev** [directus/directus#3374]
I get this error on creating Collection.

```
{
  "errors": [
    {
      "message": "insert into \"directus_fields\" (\"collection\", \"field\", \"hidden\", \"interface\", \"readonly\") values ($1, $2, $3, $4, $5) - duplicate key value violates unique constraint \"directus_fields_pkey\"",
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR"
      }
    }
  ]
}
```

and this text on interface. 

` insert into "directus_fields" ("collection", "field", "hidden", "interface", "readonly") values ($1, $2, $3, $4, $5) - duplicate key value violates unique constraint "directus_fields_pkey"`

**Comment by rijkvanzanten** [comment]
@iamvaliyev Did you manually alter the increment values of the directus_fields table by chance? The table uses auto incremented primary keys, so shouldn't just start creating duplicate keys out of the blue 🤔

**Comment by iamvaliyev** [comment]
> @iamvaliyev Did you manually alter the increment values of the directus_fields table by chance? The table uses auto incremented primary keys, so shouldn't just start creating duplicate keys out of the blue 🤔

no i added data from other database with copy paste after i get this error.

**Comment by rijkvanzanten** [comment]
@iamvaliyev Make sure to reset the auto increment sequence to match the new highest ID that you pasted in, otherwise the database will generate the next ID to be one that already exists 👍

**Comment by iamvaliyev** [comment]
> @iamvaliyev Make sure to reset the auto increment sequence to match the new highest ID that you pasted in, otherwise the database will generate the next ID to be one that already exists 👍

Thanks, After import the dump file, it is also impossible to go to the admin panel, it gives an error: Unexpected error :/

**Comment by rijkvanzanten** [comment]
Feel free to open a new bug report 👍

**Comment by benhaynes** [comment]
Unless I'm mistaken, SQL should never have collisions when the PK column is set to auto-increment. If you import from an existing dataset, I would confirm that the legacy and new column was indeed set to the SQL native auto-increment...

![Screen Shot 2020-12-10 at 11 09 34 AM](https://user-images.githubusercontent.com/522079/101797670-4af74c80-3ad8-11eb-8291-6661ce069391.png)

**Comment by rijkvanzanten** [comment]
> SQL should never have collisions when the PK column is set to auto-increment.

The trick is whether or not you're importing it with or without the original primary key. You can still import rows with values like `id=2051` even with auto increment enabled. However, this does cause the database to keep "incrementing" new rows from where it last left off, eventually running into a conflict.

**Comment by benhaynes** [comment]
Maybe that happens on certain databases, but that has never been my experience, unless my DB admin tool (Sequel Pro) is doing something in the background.

1. I created a table with an auto-increment PK
2. Added one item manually (PK: `1`)
3. Then ran a manual import with an item containing a PK of `100`
4. Confirmed that the table automatically updated the auto-increment value to `101`
5. Manually added another item, which auto-created a PK of `101`

The only way I can see this happening is if you tried to import records with a PK that already existed in the table — I've never seen a PK conflict when letting the DB set the AI-PK. 🤔
